courses/models.py

```diff
@@ -115,25 +116,25 @@ def university_name(self):
     def start_date_display(self):
         if not self.start_date:
             return ''
-        return self.start_date.strftime(ugettext(u'%b %d %Y'))
+        return localize_date(self.start_date, ugettext(u'b d Y'))
 
     @property
     def end_date_display(self):
         if not self.end_date:
             return ''
-        return self.end_date.strftime(ugettext(u'%b %d %Y'))
+        return localize_date(self.end_date, ugettext(u'b d Y'))
 
     @property
     def enrollment_start_date_display(self):
         if not self.enrollment_start_date:
             return ''
-        return self.enrollment_start_date.strftime(ugettext(u'%b %d %Y'))
+        return localize_date(self.enrollment_start_date, ugettext(u'b d Y'))
 
     @property
     def enrollment_end_date_display(self):
         if not self.enrollment_end_date:
             return ''
-        return self.enrollment_end_date.strftime(ugettext(u'%b %d %Y'))
+        return localize_date(self.enrollment_end_date, ugettext(u'b d Y'))
```

Comment:
Vu qu'on a quatre propriétés au code identique, est-ce que ça ne serait pas le bon moment pour unifier tout ça : 

    def enrollment_end_date_display(self):
        return localized_date(self.enrollment_end_date)

    def localized_date(date_time):
         if not date_time:
             return ''
        return localize_date(date_time, ugettext(u'b d Y'))